server/game/cards/06-CotE/DefendYourHonor.js

```diff
@@ -0,0 +1,49 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes, DuelTypes, Players } = require('../../Constants');
+
+class DefendYourHonor extends DrawCard {
+    setupCardAbilities() {
+        this.interrupt({
+            title: 'Initiate a military duel',
+            when: {
+                onInitiateAbilityEffects: (event, context) =>
+                    context.game.isDuringConflict() &&
+                    event.card.type === CardTypes.Event && context.player.opponent
+            },
+            targets: {
+                challenger: {
+                    cardType: CardTypes.Character,
+                    controller: Players.Self,
+                    cardCondition: (card) => card.isParticipating()
+                },
+                duelTarget: {
+                    dependsOn: 'challenger',
+                    cardType: CardTypes.Character,
+                    controller: Players.Opponent,
+                    cardCondition: card => card.isParticipating()
+                }
+            },
+            effect: 'initiate a military duel between {1} and {2}',
+            effectArgs: (context) => [context.targets.challenger, context.targets.duelTarget],
+            handler: context => {
```

Comment:
Why is this a handler not a duel game action?

Reply:
Short answer is it didn't work as a game action.  I didn't look too deep into it, but `wouldInterrupt` would not trigger when I used a gameAction at all.  It looks like all the `wouldInterrupt`s only use `handler` so I assume that it is not setup to handle `gameAction`.

Any help to get that working would be much appreciated.